File: python/03-repeticao/main048.py
''' Faça um programa que leia 7 números inteiros e no final mostre o somatório 
entre eles.


Data 16/10/2025 -> 24ª Dia
'''

somatorio = 0
cont = 1
while cont <= 7:
    num = int(input('Digite um número: '))
    somatorio += num
    cont += 1

print(somatorio)

File: c/03-repeticao/main048.c
/*Faça um programa que leia 7 números inteiros e no final mostre o somatório 
entre eles.


Data 16/10/2025 -> 24ª Dia
*/

#include <stdio.h>

int main() {

    int somatorio = 0,num;

    for(int i = 1; i <= 7; i++) {

        printf("Digite um numero: ");
        scanf("%d",&num);
        somatorio += i;
    }
    printf("%d",somatorio);
    return 0;
}